falcon/http_error.py

```diff
@@ -51,7 +52,7 @@ class HTTPError(Exception):
         status (str): HTTP status code and text, such as "400 Bad Request"
 
     Keyword Args:
-        title (str): Human-friendly error title (default ``None``).
+        title (str): Human-friendly error title.
```

Comment:
We should explain that this arg is optional and if not passed, will default to the status.